test/manifests/multiperiod-dynamic.js

```diff
@@ -0,0 +1,953 @@
+export const parsedManifest = {
+  allowCache: true,
+  discontinuityStarts: [],
+  segments: [],
+  endList: true,
```

Comment:
so, this is outputting as `true` from the parser but other `endList`s are outputting as false. Not sure which is current for a dynamic playlist but the values in here are currently set to have the test passing.

Reply:
Ok, from an offline conversation, this is currently expected as it is currently hardcoded to be true (https://github.com/videojs/mpd-parser/blob/master/src/toM3u8.js#L256-L260). As long as individual playlist and media group items are false, it's fine.